template/nuxt-app/assets/definitions.styl

```diff
@@ -28,6 +28,10 @@ btn-pad-h = 30px
 btn-pad-h-min = 30px
 btn-pad-h-small = 20px
 
+// input sizing
+input-h = 45px
+input-h-min = 45px
```

Comment:
@mattaebersold what do you think of defaulting these to `btn-desktop-h` and `btn-mobile-h` so, by default, they'd be the same height.  Thoughts?